README.md

```diff
@@ -35,87 +48,58 @@ npm start -- --client-id=main
 Configurations can also be passed via the command line or environment variables prefaced
 with `BPANEL_`. Read more about configuring bPanel [here](#configuration).
 
-### Quick Start w/ Docker
+### With Docker
+
 From inside the project directory in your terminal, run:
 
 ```bash
 npm install && docker-compose up -d
 ```
 
-This will start up three docker services as background daemon processes.
+This will start up three docker services as [detached processes running in the background](https://docs.docker.com/compose/reference/up/).
```

Comment:
Services are a thing in docker - https://docs.docker.com/get-started/part3/#about-services

We aren't specifically spinning up services, but containers. Maybe:

`this will start up docker containers as...`